provisioning/packer/includes/Vagrantfile

```diff
@@ -174,3 +174,7 @@ if cwd != nil && Digest::MD5.file("#{cwd}/#{vagrantfile}").hexdigest == "ef550c4
     end
   end
 end
+
+# Load local Vagrantfile, if exists.
+include_vagrantfile = File.expand_path('../Vagrantfile.local', __FILE__)
```

Comment:
@Decipher this won't work in the pre-packaged Vagrantfile as `File.expand_path` will expand to the box path.